jicofo-selector/src/main/kotlin/org/jitsi/jicofo/bridge/colibri/ColibriV2SessionManager.kt

```diff
@@ -366,6 +366,16 @@ class ColibriV2SessionManager(
         }
 
         synchronized(syncRoot) {
+            // We may have already removed the session and/or participant, for example due to a previous failure. In
+            // that case we shouldn't act on this error (hence removeBridge=false).
+            if (!sessions.containsValue(session)) {
+                logger.info("Ignoring response for a session that's no longer active.")
```

Comment:
Maybe include the session name in the log message and/or exception?  Similarly for participant below.